src/main/java/xyz/vegaone/easytracking/domain/Feature.java

```diff
@@ -0,0 +1,48 @@
+package xyz.vegaone.easytracking.domain;
+
+import lombok.Getter;
+import lombok.Setter;
+import xyz.vegaone.easytracking.utils.Status;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Getter
+@Setter
+@Entity
+public class Feature {
+
+    @Id
+    @GeneratedValue
+    @Column(name = "id")
+    private Long id;
+
+    @Column(name = "title")
+    private String title;
+
+    @Column(name = "description")
+    private String description;
+
+    @Column(name = "status")
+    private Status status;
+
+    @Column(name = "epic")
+    @ManyToOne
+    private Epic epic;
+
+    @Column(name = "userstory")
+    @OneToMany(mappedBy = "feature")
+    private List<UserStory> userStories;
```

Comment:
I'm in favor of using `userStoryList` as oposed to `userStories`. IMO it's easier to spot the difference between 
`userStory` and `userStoryList` than between 
`userStory` and `userStories`

Would you mind switching to `userStoryList`?